geopandas/io/tests/test_arrow.py

```diff
@@ -987,3 +994,286 @@ def test_read_parquet_geoarrow(geometry_type):
+        read_parquet(basedir, bbox=(0, 0, 20, 20))
+
+
+def test_check_bbox_covering_column_in_parquet(tmpdir, naturalearth_lowres):
+    # check error message
+    from pyarrow import parquet
+
+    df = read_file(naturalearth_lowres)
+    filename = os.path.join(str(tmpdir), "test.pq")
+    df.to_parquet(filename)
+    metadata = parquet.read_schema(filename).metadata
+    geo_metadata = _decode_metadata(metadata.get(b"geo", b""))
+    with pytest.raises(ValueError, match="No covering bbox in parquet file."):
+        _validate_bbox_column_in_parquet(geo_metadata)
+
+
+def test_convert_bbox_to_parquet_filter():
+    # check conversion of bbox to parquet filter expression
+    import pyarrow.compute as pc
+
+    bbox = (0, 0, 25, 35)
+    expected = ~(
+        (pc.field(("bbox", "xmin")) > 25)
+        | (pc.field(("bbox", "ymin")) > 35)
+        | (pc.field(("bbox", "xmax")) < 0)
+        | (pc.field(("bbox", "ymax")) < 0)
+    )
+    assert expected.equals(_convert_bbox_to_parquet_filter(bbox, "bbox"))
+
+
+def test_read_parquet_bbox_column_default_behaviour(tmpdir, naturalearth_lowres):
+    # check that bbox column is not read in by default
+
+    df = read_file(naturalearth_lowres)
+    filename = os.path.join(str(tmpdir), "test.pq")
+    df.to_parquet(filename, write_covering_bbox=True)
+    pq_df_default = read_parquet(filename)
+    assert "bbox" not in pq_df_default
+
+    pq_df_read_bbox = read_parquet(filename, read_bbox_column=True)
+    assert "bbox" in pq_df_read_bbox
+
+
+def test_read_parquet_colums_and_bbox(tmpdir, naturalearth_lowres):
+    # confirm columns list over-rides read_bbox_column argument.
+    # if column list includes 'bbox' and read_bbox_column=False -> include 'bbox'
+    # if column list excludes 'bbox' and read_bbox_column=True -> exclude 'bbox'
+    from pyarrow import parquet
+
+    df = read_file(naturalearth_lowres)
+    filename = os.path.join(str(tmpdir), "test.pq")
+    df.to_parquet(filename, write_covering_bbox=True)
+    schema = parquet.read_schema(filename)
+    columns = schema.names
+
+    pq_df1 = read_parquet(filename, columns=columns, read_bbox_column=False)
+    assert "bbox" in pq_df1
+
+    columns.remove("bbox")
+    pq_df2 = read_parquet(filename, columns=columns, read_bbox_column=True)
+    assert "bbox" not in pq_df2
+
+
+def test_filters_format_as_DNF(tmpdir, naturalearth_lowres):
+    df = read_file(naturalearth_lowres)
+    filename = os.path.join(str(tmpdir), "test.pq")
+    df.to_parquet(filename, write_covering_bbox=True)
+
+    filters = [("gdp_md_est", ">", 20000)]
+
+    pq_df = read_parquet(filename, filters=filters, bbox=(0, 0, 20, 20))
+    assert pq_df["name"].values.tolist() == [
+        "Dem. Rep. Congo",
+        "France",
+        "Nigeria",
+        "Cameroon",
+        "Ghana",
+        "Algeria",
+        "Libya",
+    ]
+
+
+def test_filters_format_as_expression(tmpdir, naturalearth_lowres):
```

Comment:
Maybe parametrize the above and this one on a `filters` param, as I think that's the only thing that differs here? (to avoid some duplication)